scripts/generate-app/templates/client/legacy/index.js/official.tpl

```diff
@@ -15,4 +15,5 @@ require('mano-legacy/hash-nav-ordered-list-controls');
 require('mano-legacy/hash-nav-modal');
 require('mano-legacy/radio-match');
 require('mano-legacy/checkbox-toggle');
+require('mano-legacy/get-text-child');
```

Comment:
Yes, forgot to say that I will wait for confirmation on this one before proceeding with adapts.